airflow/providers/databricks/hooks/databricks.py

```diff
@@ -526,7 +530,7 @@ def update_repo(self, repo_id: str, json: dict[str, Any]) -> dict:
         :param json: payload
         :return: metadata from update
         """
-        repos_endpoint = ("PATCH", f"api/2.0/repos/{repo_id}")
+        repos_endpoint = UPDATE_REPO_ENDPOINT[1].format(repo_id=repo_id)
```

Comment:
It looks like you use only the second part of the tuple and not the whole tuple which is needed.

Reply:
Yeah, it looks like you are setting repos endpoint to just the endpoint will which not work for [self._do_api_calls ](https://github.com/apache/airflow/blob/24394ee79df085edf8502894ceb82aa0150147bf/airflow/providers/databricks/hooks/databricks_base.py#L552-L572) as it is expecting a tuple